ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -111,6 +150,70 @@ extension UIColor {
         }
     }
 
+    @objc public var light: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .light) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var lightHighContrast: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .light,
+                                             accessibilityContrast: .high) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var lightElevated: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .light,
+                                             userInterfaceLevel: .elevated) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var lightElevatedHighContrast: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .light,
+                                             accessibilityContrast: .high,
+                                             userInterfaceLevel: .elevated) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var dark: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .dark) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var darkHighContrast: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .dark,
+                                             accessibilityContrast: .high) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var darkElevated: UIColor {
+        guard let color = resolvedColorValue(userInterfaceStyle: .dark,
+                                             userInterfaceLevel: .elevated) else {
+            return self
+        }
+        return UIColor(colorValue: color)
+    }
+
+    @objc public var darkElevatedHighContrast: UIColor {
```

Comment:
curious why exposing these these colors property was important on this pr?

Reply:
Many components create custom color combinations by picking and choosing light and dark variants from other colors. For example, NavigationBar uses a colorful light color and a neutral dark color. To do that, we need to expose the specific variants of a given color to help with the deconstruction and reconstruction.